### PR TITLE
Add microsoft-dynamics365-configuration.md

### PR DESCRIPTION
**Pull request by KasunAratthanage**

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+# Setting up the Microsoft Dynamics365 Environment with Azure
+
+The Microsoft Dynamics 365 (Microsoft Dynamics CRM) Connector allows you to access the [Microsoft Dynamics 365 Web API](https://docs.microsoft.com/en-us/previous-versions/dynamicscrm-2016/developers-guide/mt593051(v=crm.8)?redirectedfrom=MSDN) through WSO2 ESB. The Microsoft Dynamics CRM system (now known as Microsoft Dynamics 365) is a standalone CRM product from Microsoft that provides sales, marketing, and service management capabilities only via individual modules.
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md]: WSO2 ESB >> WSO2 Enterprise Integrator

File: en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    
+    11. Before exiting the user form, choose MANAGE ROLES and assign a security role to this application user so that the application user can access the desired organization data.
+   
+## Generate Access Token 
+
+After setup Azure and Microsoft Dynamics 365 CRM configurations you can get access token by invoking the following HTTP request.
+
+POST URL:
+
+https://login.microsoftonline.com/<tenant id> /oauth2/token
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md]: What is `tenant id`? 
When you have `<` and `>` how does it appear in the docs?

File: en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+| Key        | Value |
+| ------------- |-------------|
+| client_id    | Application ID of the registered app in Azure. |
+| resource      |https://trial.crm.dynamics.com (Dynamics 365 Online Insance URL |
+| Client_secret |Key value from the registered app in Azure|
+| Grant_type    |client_credentials|
+
+Making a Request using Postman
+
+<img src="/assets/img/connectors/MSdynamics365-access-token.png" title="Obtaining access token" width="800" alt="Obtaining access token"/>
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md]: Need to mention that you want to  save which fields to proceed with configuring the connector.

File: en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md]: Changed.

File: en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md]: In the above steps already describe how to get the Tenant id.

File: en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md (hunk 3 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md]: Added.

File: en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md]: Using "<>" in the docs need to verify.

**Comment by KasunAratthanage** [comment]
Close this PR. This content included in following pull request.
https://github.com/wso2/docs-ei/pull/1224

**Pull request closed without merging**